views/measure-sidebar.js

```diff
@@ -15,6 +15,9 @@ module.exports = (state, dispatch) => {
   const measureUrl = l.author_username
     ? `/${l.author_username}/${l.type === 'nomination' ? 'nominations' : 'legislation'}/${l.short_id}`
     : `/${l.type === 'nomination' ? 'nominations' : 'legislation'}/${l.short_id}`
+  const hideTargetReps = (l) => (
```

Comment:
Same comment as above... `hideTargetReps` is a weird name for here... there is no `targetReps` component that is being hidden, unlike in the other file.